client/data/transactions/resolvers.js

```diff
@@ -19,6 +19,17 @@ import {
 } from './actions';
 import { formatDateValue } from 'utils';
 
+function getTimeZone() {
```

Comment:
```suggestion
function getUserTimeZone() {
```

To avoid any ambiguity with the blog's timezone